packages/contracts-bedrock/contracts/test/L1StandardBridge.t.sol

```diff
@@ -39,7 +39,9 @@ contract L1StandardBridge_Receive_Test is Bridge_Initializer {
         assertEq(address(op).balance, 0);
 
         vm.expectEmit(true, true, true, true);
-        emit ETHBridgeInitiated(alice, alice, 100, hex"");
+        emit ETHDepositInitiated(alice, alice, 100, hex"");
+        // Below should NOT be emitted since this is emitted in the StandardBridge receive and not the L1StandardBridge receive.
+        // emit ETHBridgeInitiated(alice, alice, 100, hex"");
```

Comment:
Did you mean to leave this line commented out?

Reply:
Yes, but removed since it's documented in-line.